DemoApp/AppConfigViewController/AppConfigViewController.swift

```diff
@@ -0,0 +1,274 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// The Demo App Configuration.
+struct DemoAppConfig {
```

Comment:
While this is really nice, I feel like it should have been in a separate PR. There's no need to split this one, but let's try do take it into account for the future 🙏

Reply:
Yes, I agree, but usually what happens is that these "optional" PRs usually do not get much attention if they are not absolutely necessary. And I wanted to make sure this goes in since without it, this PR wouldn't be possible to QA 👍